### DISCUSSION TITLE
Chaining multiple discriminators

### DISCUSSION THREAD

**Discussion by ThatOneAwkwardGuy** [mikro-orm/mikro-orm#2015] [Q&A] [answered]
Hi all,

Just wanted to know if it was possible to chain discriminators together?

e.g

- BaseAsset (abstract) (type as discriminant)
  - PropertyAsset (type => property)
  - BaseBankingAsset (type=> banking, source as discriminant)
    - ManualBankingAsset (type=> banking, source=>manual)
    - AutoBankingAsset (type=>banking, source=>auto)

**Comment by B4nan** [comment, marked as answer]
yes you can, that is exactly how the ORM tests look like:

https://github.com/mikro-orm/mikro-orm/blob/master/tests/features/single-table-inheritance.mysql.test.ts

check the `CompanyOwner2/Manager2/Employee2/BaseUser2` entities

**Reply by ThatOneAwkwardGuy** [reply]
> yes you can, that is exactly how the ORM tests look like:
> 
> https://github.com/mikro-orm/mikro-orm/blob/master/tests/features/single-table-inheritance.mysql.test.ts
> 
> check the `CompanyOwner2/Manager2/Employee2/BaseUser2` entities

sorry, I wrote the list slightly wrong, BaseAsset has a discriminant for type and BaseBankingAsset has a discriminant on source

**Reply by B4nan** [reply]
nope, that's not possible, there can be only one discriminator column

**Reply by ThatOneAwkwardGuy** [reply]
> nope, that's not possible, there can be only one discriminator column

Thanks for the response, kinda felt I was overdoing it anyway, just needed to know haha
